docs/guides/external-node/prepared_configs/mainnet_consensus_config.yaml

```diff
@@ -0,0 +1,10 @@
+server_addr: '0.0.0.0:3054'
+public_addr: '<your public IP goes here>:3054'
+max_payload_size: 5000000
```

Comment:
Any guess for a good `max_batch_size`? As per Roman's advise the zksync-era node will multiply it by 5000 if it's missing, which would be huge.

Reply:
this is a good question, there are additional constraints on batch that might allow us to make it smaller than just max block size * max blocks per batch. This is a larger topic, let's leave it out of scope of gossipnet announcement.